src/components/Icon/Icon.css

```diff
@@ -45,3 +45,7 @@
   width: 1rem;
   top: 0.1rem;
 }
+
+.y-avatar--badge > .y-icon {
+  top: 0;
+}
```

Comment:
Moved this here for now. I think this needs to be addressed with https://github.com/Microsoft/YamUI/issues/327

Reply:
sounds good. could you add a comment above it referencing #327?